pkg/core/resources/apis/mesh/mesh_validator.go

```diff
@@ -253,3 +254,16 @@ func validateMetrics(metrics *mesh_proto.Metrics) validators.ValidationError {
 	}
 	return verr
 }
+
+func validateZoneEgress(routing *mesh_proto.Routing, mtls *mesh_proto.Mesh_Mtls) validators.ValidationError {
+	var verr validators.ValidationError
+	if routing == nil {
+		return verr
+	}
+	if routing.ZoneEgress {
+		if mtls == nil {
```

Comment:
check `mtls#enabledBackend`. You can define mTLS but not specify `enabledBackend`